packages/discord-utilities/src/lib/InteractionOptionResolver.ts

```diff
@@ -80,7 +80,7 @@ export class InteractionOptionResolver {
 		}
 	}
 
-	public get(name: string, required?: boolean | false): APIApplicationCommandInteractionDataOption | null;
+	public get(name: string, required?: boolean): APIApplicationCommandInteractionDataOption | null;
```

Comment:
Those were supposed to be this:
```suggestion
	public get(name: string, required?: false): APIApplicationCommandInteractionDataOption | null;
```

But maybe we can use discord.js's `If` type just like /result does, to simplify and improve the overloads.